src/Graphics/src/Graphics/Platforms/MaciOS/PlatformCanvas.cs

```diff
@@ -657,19 +670,15 @@ private void DrawImageCallback(CGContext context)
 #else
 				var cgImage = platformImage.CGImage;
 #endif
```

Comment:
This draws the image but inversed in view. The inverse logic is also handled before this method is called. So removed the duplication. Referred DrawPatternCallback().